public/css/style.css

```diff
@@ -842,12 +844,33 @@ body * {
     
 }
 
-.source-export,
+.source-export{
+    font-size:0.60em;
+    text-decoration:none;
+    color:#777;
+    margin-left:10px;
+    background: url(images/export.png) no-repeat left center #65a06c;
```

Comment:
I am not really sure I want to add icons to imports and exports, since icons bring attention to things and I feel these action are used too rarely to warrant that.